assets/questions/isLeapYear.js

```diff
@@ -0,0 +1,196 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for isLeapYear question.
+ */
+
+globalData.questions['isLeapYear'] = {  // eslint-disable-line dot-notation
+  title: 'Identifying Leap Years',
+  starterCode: {
+    python:
+`def isLeapYear(year):
+    return False
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def returnNone(cls, unused_year):
+        return None
+
+    @classmethod
+    def returnTrue(cls, unused_year):
+        return True
+
+    @classmethod
+    def returnFalse(cls, unused_year):
+        return False
+
+    @classmethod
+    def handleDivisibleByFourCaseOnly(cls, year):
+        return year % 4 == 0
+`
+  },
+  tasks: [{
+    id: 'isLeapYear',
+    instructions: [{
+      content: [
+        'Three criteria must be taken into account to identify leap years:'
```

Comment:
The wording seems very ... formal, here. What about:

"To determine whether or not a given year is a leap year, we must consider three criteria:"

Reply:
Ah, I just followed the original question. I do prefer the updated wording, and have changed it -- thanks!